spec/services/solidus_bolt/users/sync_addresses_service_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe SolidusBolt::Users::SyncAddressesService, :vcr, :bolt_configuration do
+  subject(:sync_addresses_service) do
+    described_class.call(user: user, access_token: ENV.fetch('BOLT_ACCESS_TOKEN', 'Fake_access_token'))
```

Comment:
why don't we add the secret to the circleci config instead of providing an alternative here?

Reply:
Because we need it even in the local environment when we don't want to record the cassettes